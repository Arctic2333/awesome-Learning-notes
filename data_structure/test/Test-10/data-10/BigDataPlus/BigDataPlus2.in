9318571039857093457098357091835917920938538183487456928345035127298345834778506384769489848538012902932345673846787
834795836598758186139846298347634618923746983726498172349178234978612398476929
