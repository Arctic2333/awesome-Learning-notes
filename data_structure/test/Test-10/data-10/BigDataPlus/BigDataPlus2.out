9318571039857093457098357091835917921773334020086215114484881425645980453702253368495988020887191137910958072323716